src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1014,7 +1014,14 @@ private void AppendParameters(SeparatedSyntaxList<ParameterSyntax> parameters, S
                     builder.Append(' ');
                 }
 
-                AppendTokens(parameter.Type, builder);
+                if (parameter.Type != null)
+                {
+                    AppendTokens(parameter.Type, builder);
+                }
+                else
+                {
+                    builder.Append(parameter.Identifier.Text);
```

Comment:
❔ Should this be `ValueText`? Doesn't seem like it matterns for the `__arglist` case (since you can't write `_\u005farglist` for example), but I can't say for sure that it doesn't make a difference elsewhere.